dvc/output.py

```diff
@@ -1138,6 +1187,7 @@ def is_plot(self) -> bool:
     Output.PARAM_PLOT: bool,
     Output.PARAM_PERSIST: bool,
     Output.PARAM_CHECKPOINT: bool,
+    Output.PARAM_TREE: [TREE_ENTRY_SCHEMA],
```

Comment:
Is the idea to keep this in `.dvc` file? What do you guys think of an index file that is kept in a remote? It'd be like our cache directory, but only `.dir` file equivalent? That way, we can only have the path to the index here.

Reply:
Yes, this should be kept in `.dvc` file for now and also in `dvc.lock` for pipeline outputs now or later. The idea is to not have to have any remotes or caches.

Reply:
What’s the main requirement here though? Not having cache or having a human readable folder structure?

Reply:
Not having a cache.